tests/test_decoder.py

```diff
@@ -714,17 +714,17 @@ def test_end_comment():
 
 
 def test_parse_error():
-    with pytest.raises(pvl.decoder.ParseError):
```

Comment:
But is `foo=` actually a valid label according to the PVL specification?

http://pirlwww.lpl.arizona.edu/software/PPVL/CCSDS-641.0-B-2.pdf

I don't think so.  So I think the `pvl` module needs to have two modes, a strict mode where it actually raises an error for truly bad labels and another mode where it is permissive and does what you're doing.  `mode='strict'` and `mode='permissive'`.

The default should be `strict` and the error message raised on error should be unique and prominent enough in the docs that when users google it, it shows up in our docs and then can then switch to `mode='permissive'`.